Bottom: df93c501d6eda8afecfc7e5a466da2223351a067
Top:    866bd7287bf3b7e46f8327be46acee87b6e8bba0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-08 02:12:20 +0000

Refresh of convet-some-safe-usage-of-myst

---

diff --git a/src/copy.c b/src/copy.c
index c8244ed..ee831b9 100644
--- a/src/copy.c
+++ b/src/copy.c
@@ -220,7 +220,7 @@ CopyLine (LayerTypePtr Layer, LineTypePtr Line)
   if (!line)
     return (line);
   if (Line->Number)
-    line->Number = MyStrdup (Line->Number, "CopyLine");
+    line->Number = strdup (Line->Number);
   DrawLine (Layer, line, 0);
   AddObjectToCreateUndoList (LINE_TYPE, Layer, line, line);
   return (line);
